README.md

```diff
@@ -14,17 +14,17 @@ Let’s inspect the rhpam-7-businesscentral cekit module:
 schema_version: 1
 name: "rhpam-7-businesscentral"
 version: "1.0"
-description: "Red Hat Business Central 7.13 installer"
+description: "IBM Business Automation Manager Open Editions Business Central 7.13 installer"
```

Comment:
we could use only replacement of Red Hat with IBM i think.
Where is the full name use BAMOE but otherwise it is good to replace only necessary refs/names.

Reply:
sounds good.